.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "up-core-api"]
 	path = up-core-api
 	url = https://github.com/eclipse-uprotocol/up-core-api.git
+	tag = uprotocol-core-api-1.5.7
```

Comment:
It appears the tag attribute works and if i do not put this in and someone calls git submodules update, it will fetch the latest again and override the 1.5.7 version that we need to pull so I don't see an alternative.

Reply:
Backed out the change per conversation over chat.